### PR TITLE
[Crashlytics ODF] update release notes

### PR DESCRIPTION
**Pull request by themiswang**
### Description
Update release notes for upcoming release

### CODE REVIEW

File: docs/readme.md [hunk 1]
```diff
@@ -70,6 +70,7 @@ Release Notes
 ### Upcoming Release
 - Changes
     - Analytics: Added `SetConsent()` and `GetSessionIdAsync()` APIs.
+    - Crashlytics: Add `ReportUncaughtExceptionsAsFatal` propertie and `LogExceptionAsFatal` API. Crashlytics now support on-demand fatal reports.
```
Comment (themiswang) [inline docs/readme.md:73]: Hi @rachelsaunders, can you help to review this release notes? Apparently unity also automated their release notes generation.

Comment (rachelsaunders) [inline docs/readme.md:73]:
```suggestion
    - Crashlytics: Add `ReportUncaughtExceptionsAsFatal` property and `LogExceptionAsFatal` API. With these changes, Crashlytics now supports on-demand fatal reports.
```

**Review by rachelsaunders — CHANGES_REQUESTED** [review]

File: docs/readme.md (hunk 1 above)
Comment (rachelsaunders) [reply docs/readme.md]: Also, I don't know if the style for Unity release notes use the verb form of "Add" vs "Added"?

Comment (themiswang) [reply docs/readme.md]: From here I can see they are using `Add` a lot for new features: https://firebase.google.com/support/release-notes/unity#10.2.0

Comment (themiswang) [reply docs/readme.md]: Done for the change!

**Review by a-maurice — APPROVED** [review]
Just wondering, is there going to be a guide or some equivalent added to the firebase site to describe what this actually means?

**Comment by themiswang** [comment]
> Just wondering, is there going to be a guide or some equivalent added to the firebase site to describe what this actually means?

Yes, we are devrel plan to add guide on `get start` and `customize your crash report` page, for reference: https://docs.google.com/document/d/1zKV62UMl_7DDfiFNIjTlGxfyTI2PRhTwOpEhsb1E9eg/edit?resourcekey=0-ujEJR4kFnUpXUfCIDkhMdw# @mrober is working on the draft for it!

File: docs/readme.md (hunk 1 above)
Comment (themiswang) [reply docs/readme.md]: Removed notes for support of ODF.

**Review by rachelsaunders — APPROVED** [review]

**Pull request merged**